drivers/include/si1145.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2016 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_si1145 SI1145
```

Comment:
I many be very nitpicky about the naming, but according to the datasheet it is Si1145 instead of SI1145 :-)